components/button/style.scss

```diff
@@ -155,6 +155,7 @@
 	}
 
 	&:disabled {
+		cursor: default;
```

Comment:
Are these styles for `&.is-primary` and `&.is-default` variations needed now that we have added the `cursor` style on the base `.components-button:disabled` ?

Reply:
Why above do we have both `&[disabled]` and `&:disabled`, but not here?

Reply:
Probably not, and it was like that when I got here.

I'll take the time to tidy it up 👍